app/src/main/res/values/strings.xml

```diff
@@ -2,5 +2,10 @@
     <string name="app_name">JavaBaseProject</string>
     <string name="activity_main_button_text">PRESS ME</string>
     <string name="activity_main_title_text">- AVAILABLE PARKING SPOTS -</string>
-    <string name="activity_main_available_text">AVAILABLE SPOTS: </string>
+    <string name="activity_main_available_text">AVAILABLE SPOTS: %s</string>
+    <string name="fragment_dialog_set_available_text">SET AVAILABLE SPOTS</string>
+    <string name="fragment_dialog_accept_button_text">ACCEPT</string>
+    <string name="fragment_dialog_cancel_button_text">CANCEL</string>
+    <string name="toast_exceeded_int">USE A VALUE LOWER THAN 2,147,483,647</string>
```

Comment:
remove this please, that value is not a viable option for a parking lot

Reply:
Done.